docs/en/serverless/logging/monitor-datasets.mdx

```diff
@@ -0,0 +1,51 @@
+<p><DocBadge template="beta" /></p>
+
+The **Data Set Quality** page provides an overview of your log data sets.
+Use this information to get an idea of your overall log data set quality and find data sets that contain incorrectly parsed documents.
+Access the Datasets page from the main ((kib)) menu and go to **Stack Management** → **Data Set Quality*.
+
+{/* Update screenshot when changes are done <DocImage size="2" url="../images/logs-dataset-overview.png" alt="Screen capture of the data set overview" /> */}
+
+<DocCallOut title="Requirements">
+  Users with the `viewer` role can see the Datasets Quality summary. To see the Active Datasets and Estimated Data summaries, users need the `monitor` [index privilege](((ref))/security-privileges.html#privileges-list-indices) for the `logs-*-*` index.
+</DocCallOut>
+
+The quality of your data sets is based on the percentage of degraded documents in each data set.
+A degraded document in a data set contains the [`_ignored`](((ref))/mapping-ignored-field.html) property because one or more of its fields were ignored during indexing.
+Fields are ignored for a variety of reasons.
+For example, when the [`ignore_malformed`](((ref))/mapping-ignored-field.html.html) parameter is set to true, if a document field contains the wrong data type, the malformed field is ignored and the rest of the document is indexed.
+
+From the data set table, you'll find information for each data set such as its namespace, size, when the data set was last active, as well as the percentage of degraded docs.
+The percentage of degraded documents determines the data set's quality according to the following scale:
+
+* Good (<DocImage flatImage alt="Good icon" url="../images/green-dot-icon.png" />): 0% of the documents in the data set are degraded.
+* Degraded (<DocImage flatImage alt="Degraded icon" url="../images/yellow-dot-icon.png" />): Greater than 0% and up to 3% of the documents in the data set are degraded.
+* Poor (<DocImage flatImage alt="Poor icon" url="../images/red-dot-icon.png" />): Greater than 3% of the documents in the data set are degraded.
+
+Opening the details of a specific data set shows the degraded documents history, a summary for the data set, and other details that can help you determine if you need to investigate any issues.
+
+### Investigate issues in individual data sets
+To investigate issues in data sets with poor or degraded quality:
+
+1. Find data sets with degraded documents using the **Degraded Docs** column of the data sets table.
+1. Click the percentage in the **Degraded Docs** column to open the data set in Logs Explorer.
+
+The **Documents** table in Logs Explorer is automatically filtered to show documents that were not parsed correctly.
+Under the *actions* column, you'll see the degraded document icon (<DocIcon type="indexClose" title="degraded document icon" />).
+
+Now that you know which documents contain ignored fields, examine them more closely to find the origin of the issue.
```

Comment:
This is being changed by @achyutjhunjhunwala at the moment.
He is creating a detailed section for the ignored fields. Please Aj, add more details here so we can have this part updated

Reply:
This might not be ready before 8.15 FF (July 2nd)